go.mod

```diff
@@ -51,7 +51,7 @@ require (
 	go.opentelemetry.io/otel/sdk v1.27.0
 	go.opentelemetry.io/otel/sdk/metric v1.27.0
 	go.opentelemetry.io/otel/trace v1.27.0
-	go.temporal.io/api v1.36.0
+	go.temporal.io/api v1.36.1-0.20240808055503-2e16191f5553
```

Comment:
Will switch to a commit on api-go master branch once the PR in the api repo is landed.